src/Microsoft.AspNetCore.ResponseCaching/ResponseCachingOptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.ComponentModel;
+using Microsoft.AspNetCore.ResponseCaching.Internal;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    public class ResponseCachingOptions
+    {
+        /// <summary>
+        /// The largest cacheable size for the response body in bytes.
+        /// </summary>
+        public int? MaximumCachedBodySize { get; set; }
```

Comment:
Should we specify a default size? Also could consider the naming `MaximumCacheableBodySize` or `MaximumBodySizeToCache`.